packages/p2p/stores/order-store.js

```diff
@@ -23,6 +23,7 @@ export default class OrderStore {
     @observable order_rerender_timeout = null;
 
     interval;
+    item_height = isMobile() ? 115 : 72;
```

Comment:
Do we still need this? `DataList` should automagically determine the height of its items.